### PR TITLE
add travis ci for sol5 test

### PR DESCRIPTION
**Pull request by ductm54**

### CODE REVIEW

File: .travis/test-part.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+# -*- firestarter: "shfmt -i 4 -ci -w %p" -*-
+
+set -euxo pipefail
+
+readonly test_part=${TEST_PART:-}
+
+case "$test_part" in
+Sol5)
+    npx buidler test --no-compile --config ./buidlerCoverageSol5.js
```
Comment (ilanDoron) [inline .travis/test-part.sh]: why use coverage here?

can use "buildler.config.js" ??

File: buidlerCoverageSol4.js [hunk 2]
```diff
@@ -7,6 +7,7 @@ module.exports = {
 
   paths: {
     sources: "./contracts",
-    artifacts: ".coverageV4Artifacts"
+    artifacts: ".coverageV4Artifacts",
```
Comment (ilanDoron) [inline buidlerCoverageSol4.js]: not sure why use coverage file?

File: .travis/test-part.sh (hunk 1 above)
Comment (ductm54) [reply .travis/test-part.sh]: i just copy this one from tst.sh, will fix

File: buidlerCoverageSol4.js (hunk 2 above)
Comment (ductm54) [reply buidlerCoverageSol4.js]: should not change this file. will remove

File: .travis/test-part.sh (hunk 1 above)
Comment (ductm54) [reply .travis/test-part.sh]: just check again use buidler.config.js will run every test. on this we only want to run sol5 test

File: .travis/test-part.sh [hunk 3]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+# -*- firestarter: "shfmt -i 4 -ci -w %p" -*-
+
```
Comment (ilanDoron) [inline .travis/test-part.sh]: what does this comment mean?

Comment (ductm54) [reply .travis/test-part.sh]: i use shfmt to format bash. could remove if needed

**Pull request merged**